docs/config.toml

```diff
@@ -105,3 +105,8 @@ sidebar_menu_foldable = true
 sidebar_menu_truncate = 500
 ul_show = 1
 navbar_logo = true
+
+[params.ui.feedback]
+enable = true
+yes = '<b>Glad to hear it!</b> Please <a href="https://github.com/radius-project/docs/issues/new/choose">tell us how we can improve</a>.'
```

Comment:
Do we want the call-to-action here to be opening an issue? Or is there something else we can ask the user to do? Maybe "Glad to hear it! Please make sure to star our repo if you haven't done so yet so stay up to date with the project."

cc/ @willtsai

Reply:
I think for Dapr we kept the same call to action in case people wanted to report an issue for additional feedback regardless. However, I agree that the messaging should be different, and maybe point them to engage further with our community. How about:

`'<b>Glad to hear it!</b> Please feel free to <a href="https://github.com/radius-project/radius">star our repo</a> and join our  <a href=""https://aka.ms/radius/discord"">Discord server</a> to stay up to date with the project.'`

Reply:
Let me know if this works.

Reply:
Love the first part, but I'd drop the "If you would like to also....." part, the prior section is the right length

Reply:
Should I change the beginning words of that sentence or drop all together pointing the user to create an isssue?

Reply:
I recommend leaving the "thumbs down" content as it was, don't prompt them to star or engage with the community.